app/src/Cops/Core/Entity/AuthorCollection.php

```diff
@@ -24,10 +24,10 @@ class AuthorCollection extends AbstractCollection
      *
      * @return $this
      */
-    public function findByFirstLetter($letter)
+    public function findByFirstLetter($letter, $addletters)
```

Comment:
maybe add array as parameter and empty a default value and change phpdoc accordingly to avoid scrutinizer issues ;)

````
public function findByFirstLetter($letter, array $addletters = array())